app/models/user.rb

```diff
@@ -43,6 +43,10 @@ class User < ActiveRecord::Base
 
   before_save :set_token
 
+  scope :status_one, -> { where(status: 1) }
```

Comment:
Hmm, I think at this point because we've simplified the query a lot, maybe it makes sense to just use the simple .where method since it's not much more than .published anymore. What do you think? Then it's more standard rails too... It made total sense before with the longer query but maybe now not so much?

Thanks Cess!!!

Reply:
I am sorry, I am quite lost, are you suggesting we do away with all the scopes for the user_model, or just the `status_one` scope. Yeah I see the query is so much shorter now.

Reply:
Just this `status_one` scope since it's now not much shorter than the full `where()` method. Sorry i was typing from my phone!

Reply:
No worries, cool gotcha on it.